### ISSUE TITLE
Cop idea: Using `super` with Arguments

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12353]
## Description

Always use parentheses when calling `super` with arguments

https://rubystyle.guide/#super-with-args

## Describe the solution you'd like

```ruby
# bad
super name, age

# good
super(name, age)
```
